cli/sync_service/sync_service.go

```diff
@@ -129,6 +163,24 @@ func getImage(imageName string, tagName string, dc *docker.Client) error {
 	return nil
 }
 
+// remove image. Ignore error if image does not exist
+func removeImage(imageName string, tagName string, dc *docker.Client) error {
+	// get message printer
+	msgPrinter := i18n.GetMessagePrinter()
+
+	name := fmt.Sprintf("%v:%v", imageName, tagName)
+	cliutils.Verbose(msgPrinter.Sprintf("Removing docker image %v.", name))
+	if err := dc.RemoveImage(name); err != nil {
+		fmt.Printf("RemoveImageErr: %v", err)
```

Comment:
If this is a debug message, then please use cliutils.Verbose and also use msgPrinter for translation.